### ISSUE TITLE
Add a video tutorial (Any langulage) on how to use this template

### ISSUE THREAD

**Issue by sudiptob2** [sudiptob2/cf-stats#47]
I need video tutorials on how to use this template.

- [ ] sound must be clear
- [ ] No noise
- [ ] Any language accepted

**Hacktoberfest participants are welcome.**

**Comment by shubhankarsharma876** [comment]
Can we draw a video without face camera?or is it optional.

**Comment by sudiptob2** [comment]
> No face camera is necessary.
> Yes this repo has `hacktoberfest` label

If you want to create video in any language other than eng, I can create a new issue and assign it to you

**Comment by shubhankarsharma876** [comment]
> > No face camera is necessary.
> > Yes this repo has `hacktoberfest` label
> 
> If you want to create video in any language other than eng, I can create a new issue and assign it to you

Yes I would love to contribute a video in "Hindi"(one among Indian Language) language. Thanks I am glad to find this repository. You can assign me with this task.

**Comment by sudiptob2** [comment]
> > > No face camera is necessary.
> > > Yes this repo has `hacktoberfest` label
> > 
> > 
> > If you want to create video in any language other than eng, I can create a new issue and assign it to you
> 
> Yes I would love to contribute a video in "Hindi"(one among Indian Language) language. Thanks I am glad to find this repository. You can assign me with this task.

sure, will assign you a task shortly

**Comment by sudiptob2** [comment]
take a look at #48 @shubhankarsharma876

**Comment by mak650650** [comment]
Is AI generated video is accepted

**Comment by sudiptob2** [comment]
> 

can you send me a sample AI-generated tutorial video of some random project so that I can take a look ?

**Comment by AbhishekTiwari23** [comment]
I can also create a video for this. Please let me know if I can contribute.

**Comment by AbhishekTiwari23** [comment]
And I have an idea. We can also add a background template instead of just light and dark mode. Please let me know, if I can do something

**Comment by sudiptob2** [comment]
> And I have an idea. We can also add a background template instead of just light and dark mode. Please let me know, if I can do something

Thanks for stopping by :pray: If you want to implement a new SVG template for the `stat`, you are welcome. Please create PR with initial designs. If designs are well enough, I am happy to merge :blush: @AbhishekTiwari23

**Comment by AbhishekTiwari23** [comment]
> > And I have an idea. We can also add a background template instead of just light and dark mode. Please let me know, if I can do something
> 
> Thanks for stopping by 🙏 If you want to implement a new SVG template for the `stat`, you are welcome. Please create PR with initial designs. If designs are well enough, I am happy to merge 😊 @AbhishekTiwari23

Thanks, sir, I will work on that and make pr as soon as possible
